src/components/CommentList.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import CommentCard from "./CommentCard";
+
+export default function CommentList({article_id}) {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [err, setErr] = useState(null);
+
+  useEffect(() => {
+    fetch(
+      `https://nc-news-wfs6.onrender.com/api/articles/${article_id}/comments`
+    )
+      .then((res) => {
+        if (!res.ok) throw new Error("Failed to load comments.")
+        return res.json();
```

Comment:
I think I left feedback on an earlier ticket about changing this to use axios instead. It will mean you don't have to check res.ok and you can just chain on a .catch instead 👍